### ISSUE TITLE
Add the saved current frame to the Project files panel

### ISSUE THREAD

**Issue by raindropsfromsky** [OpenShot/openshot-qt#5446]
**Describe the new feature:**
When the current frame is saved, it can be used to create a poster for the edited movie.
In this use case, it is no longer needed within the current project.

But the frame can also be used to create a "freeze frame" effect: We slice a clip, save the frame, and insert that frame between the two slices in the very next step.

So, logically, the frame should be available in the **Project files** folder. But Openshot does not save a copy there.
As a result, we have to go out and hunt for that saved frame once again.

This roundabout procedure can be shortened by directly adding the captured frame in the **Project files** panel.

(This is just like how the **Split clip** dialog saves the edited clips in the **Project files** folder, so that they are immediately available for the subsequent editing operations.)

**Comment by Colorjet3** [comment]
Hello @raindropsfromsky.  I have used the Camera (save current frame) feature several times.  I have never had to hunt for it since it prompts me to save it where I want it.

Instead of using this approach to create a "freeze frame" you can use the Right click on a clip, select Time | Freeze or Freeze and Zoom.  One of the things I don't like about this is not being able to custom select a duration.  For example, I want 3 seconds.

IMHO, the Time | Freeze or Freeze and Zoom works great.  I would add the following enhancements this:
1.  Provide a way for Custom entry for time/duration.
2.  If there are clips to the right of this clip then shift everything over but however long the Freeze is set for.  For example, if I selected 2 seconds for the Freeze, then everything to the right of this clip, on the same track, would shift 2 seconds.  This should include gaps as well.

Would love to hear your comments/feedback.

**Update:**I forgot to mention that sometimes, when use the Camera to snap a single frame, I end up using in other projects.  So it is almost like OpenShot will have to prompt you as to what you want, Save it to the project files, Save it as a .png, or both.  This definitely will complicate things.  Right now, you know it becomes an external source file and you can import it back.  You could say an enhancement would be to automatically save it also in the Project Files, but some users might complain as to why it is doing that.

**Comment by raindropsfromsky** [comment]
Love your interpretation!

In general, Openshot provides some commands that are of fixed duration.
At best, two different durations are provided:  "long" and "short" (e.g. fade in/out).
In all such commands, we should have a third "custom" option to let the user enter his own value.

**Comment by Colorjet3** [comment]
Here is an example that I just submitted for the lead developer:  https://trello.com/c/yAuUkyF4/180-time-freeze-or-freeze-and-zoom-should-allow-a-custom-time-duration-entered

I like your example of fade in/out as well.  If you can provide a list of other areas (effects) that would be nice to have the ability to customize the duration, I can then make a single entry for the lead developer to review.

**I also wanted to say thank you for all your input!**.  I'll continue reviewing your submissions in the next few days and provide my input/feedback.

**Comment by raindropsfromsky** [comment]
Thanks! I know I have bombarded too many suggestions for one day! 😄

**Comment by Colorjet3** [comment]
Yeah, you sure have overloaded the system 😆 I will check into the Trello link.  It is supposed to be open to the public via the link only.  The Search option is not exposed to the Public.

Will get back to you on this.

**Comment by raindropsfromsky** [comment]
I am curious: I had actually deleted that comment! How did you get to see that?
Do you receive every edit as a new message?

I deleted that comment because I figured out a solution: Your link has a 3-digit card number embedded.
So after I click on the link to reach the Trello page, I use the filter and search for that number.

But this may not occur to other users. 
The link should pop up the target card.

**Comment by Colorjet3** [comment]
I am not sure.  It had to be a timing thing that I saw that message.

Yes, I agree that the link should pop up the target card.  Investigating.....
